README.md

```diff
@@ -106,9 +106,11 @@ Matrix arguments are NumPy arrays for dense solvers and SciPy Compressed Sparse
 - *I have a squared norm in my cost function, how can I apply a QP solver to my problem?*
   - You can [cast squared norms to QP matrices](https://scaron.info/blog/conversion-from-least-squares-to-quadratic-programming.html) and feed the result to [`solve_qp`](https://qpsolvers.github.io/qpsolvers/quadratic-programming.html#qpsolvers.solve_qp).
 - *I have a non-convex quadratic program. Is there a solver I can use?*
-  - Unfortunately most available QP solvers are designed for convex problems.
-  - If your cost matrix *P* is semi-definite rather than definite, try OSQP.
-  - If your problem has concave components, go for a nonlinear solver such as [IPOPT](https://pypi.org/project/ipopt/) *e.g.* using [CasADi](https://web.casadi.org/).
+  - Unfortunately most available QP solvers are designed for convex problems (i.e. problems for which `P` is positive semidefinite). That's in a way expected, as solving non-convex QP problems is NP hard.
+  - CPLEX has methods for solving non-convex quadratic problems to either local or global optimality. Notice that [finding global solutions](https://www.gurobi.com/wp-content/uploads/2020-01-14_Non-Convex-Quadratic-Optimization-in-Gurobi-9.0-Webinar.pdf?x58432) can be significantly slower than [finding local solutions](https://link.springer.com/chapter/10.1007/978-1-4613-0263-6_8).
```

Comment:
Can we link to CPLEX's functionality directly, e.g. in the doc, or slides as with Gurobi?

The second sentence is a pertinent point to make. Can we link it to a general discussion of global vs local optimality in non-convex QPs? E.g. in terms of complexity rather than individual methods. (Also, a link to non-paywalled material would be more accessible.) Maybe there is *e.g.* a section in Boyd & Vandenberghe we could link to?

Alternatively, I think no link would be OK too. It should be already intuitive that global optimization is harder than local, perhaps we don't want to go too deep in the FAQ.

Reply:
> Maybe there is e.g. a section in Boyd & Vandenberghe we could link to?

As far as I recall, not, they only deal with convex problems.

> It should be already intuitive that global optimization is harder than local, perhaps we don't want to go too deep in the FAQ.

In convex optimisation it's not: every local optimiser is also a global optimiser. This repo only supports convex optimisation, so when diverting from the "convex world" I think it's nice to remind some things that are lost.

Reply:
I found the following page for CPLEX: [Distinguishing between convex and nonconvex QPs](https://www.ibm.com/docs/en/icos/12.9.0?topic=qp-distinguishing-between-convex-nonconvex-qps).

Here is the corresponding suggestion:

```suggestion
  - CPLEX has methods for solving non-convex quadratic problems to [either local or global optimality](https://www.ibm.com/docs/fr/icos/22.1.0?topic=qp-distinguishing-between-convex-nonconvex-qps). Notice that finding global solutions can be significantly slower than [finding local solutions](https://link.springer.com/chapter/10.1007/978-1-4613-0263-6_8).
```

It also removes the duplicate link to Gurobi, which we have just below. That page from the CPLEX docs also explains the global/local distinction nicely.